treetime/clock_tree.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, *args, dates=None, debug=False, real_dates=True, precision_ff
         """
         super(ClockTree, self).__init__(*args, **kwargs)
         if dates is None:
-            raise ValueError("ClockTree requires date constraints!")
+            raise MissingDataError("ClockTree requires date constraints!")
```

Comment:
If this is how you want to use MissingDataError one should probably change the description of the MissingDataError to encompass missing parameters